bin/hdfs-mesos

```diff
@@ -4,4 +4,10 @@ if [ -z "$PORT0" ] ; then
   PORT0="8765"
 fi
 
-exec java -cp lib/*.jar -Dmesos.conf.path=etc/hadoop/mesos-site.xml -Dmesos.hdfs.config.server.port=$PORT0 org.apache.mesos.hdfs.Main
+if [ -z "$JAVA_HOME" ]; then
```

Comment:
Lets name this variable JAVA_CMD or JAVA_EXEC, I'm wary of a variable called JAVA